lib/server/plugins/api/index.js

```diff
@@ -118,10 +118,18 @@ exports.register = function (plugin, options, next) {
     );
   };
 
+  internals.maybeForceGzip = function (request, reply) {
+    if (request.query.gzip === 'force') {
+      request.info.acceptEncoding = 'gzip';
+    }
+    reply();
+  };
+
   plugin.route([
     {
       method: 'OPTIONS',
       path: '/_api/{p*}',
+      config: { pre: [{ method: internals.maybeForceGzip }] },
```

Comment:
Is there a way to set this globally for `plugin.route`, so I can save myself from repeating this line in every route? I couldn’t quite figure it out from the hapi docs.

cc @latentflip et.al.